build.gradle.kts

```diff
@@ -19,7 +19,7 @@ allprojects {
         google()
         maven("https://s01.oss.sonatype.org/content/repositories/snapshots/") {
             content {
-                includeGroup(Dependencies.Groups.jellyfin)
+                includeVersionByRegex(Dependencies.Groups.jellyfin, ".*", ".*-SNAPSHOT")
```

Comment:
Are you sure this won't be a bad regex because the `.` in this constant is not escaped?????????????

Reply:
yes should be fine